pkg/products/threescale/reconciler_test.go

```diff
@@ -1494,6 +1579,9 @@ func TestReconciler_useCustomDomain(t *testing.T) {
 		{
 			name: "Use custom domain true",
 			fields: fields{
+				Config: config.NewThreeScale(config.ProductConfig{
+					"CUSTOM_DOMAIN_ENABLED": "true",
```

Comment:
I am still unclear why we need to the `CUSTOM_DOMAIN_ENABLED` flag

Reply:
I set the `CUSTOM_DOMAIN_ENABLED` config value in the 3scale reconciler each time we invoke the `useCustomDomain` function. This approach seemed better than passing more parameters to the `newAlertReconciler` function.

Reply:
This was possible the best place to do this but as I mentioned earlier the `newAlertReconciler` function has access to this information through `r.installation`. There is no need to have this flag.